src/lib/services/feature-toggle-service.ts

```diff
@@ -1179,7 +1178,7 @@ class FeatureToggleService {
         featureName: string,
         projectId: string,
         newFeatureName: string,
-        replaceGroupId: boolean = true, // eslint-disable-line
```

Comment:
Not sure here... I believe this is setting `replaceGroupId` to true, but because the next param `userName` does not have a default, it doesn't matter: the compiler would force us to provide this value in all calls to this method. What worries me a bit is that comments like `// @ts-ignore` can void the compiler hint.

If we don't set it to true... then undefined is falsy by default which might change the behavior. 

Maybe it checked all usages of the method to validate the value is always provided?

Reply:
Oh, this was manual, we turned this off explicitly with the eslint-disable-line that was in place here. It's an incorrect usage of default arguments, since default arguments must come at the end of the parameter list in order to guarantee the non-default argument.

Reply:
Turned out we had a test that was failing due to this. I rearranged the arguments so the default argument indeed comes last, there were only 5 places (1 in code, 4 in tests) that actually used this, so was manageable. I also checked that enterprise didn't use the method.